ui/src/views/Tasks/TaskLog/index.jsx

```diff
@@ -37,12 +37,19 @@ export default class TaskLog extends Component {
   }
 
   render() {
-    const { classes, match, stream } = this.props;
+    const {
+      classes,
+      match,
+      stream,
+      data: { task },
+    } = this.props;
     const url = decodeURIComponent(match.params.logUrl);
     const run = this.getCurrentRun();
 
     return (
-      <Dashboard title="Log" disablePadding>
+      <Dashboard
+        title={`${task ? task.metadata.name : 'Task'} Log`}
```

Comment:
```suggestion
        title={title ? `Log ${task.metadata.name}` : 'Log'}
        disableTitleFormatting
```

Reply:
- [ ] The page doesn't load for me when you check for the title.

Reply:
@helfi92 I didn't mean to mark this as resolved. The page doesn't load for me if you check for title with `title ? Log...` and it flashes with a message that says title is not defined.

Reply:
I went ahead and made the changes with `task ?` instead of `title ?` since title doesn't exist and it causes an error.